src/Workspaces/CSharp/Portable/Recommendations/CSharpRecommendationServiceRunner.cs

```diff
@@ -143,7 +143,7 @@ private ImmutableArray<ISymbol> GetSymbolsForGlobalStatementContext()
                 position >= token.Span.End)
             {
                 var compUnit = (CompilationUnitSyntax)syntaxTree.GetRoot(_cancellationToken);
-                if (compUnit.Usings.Count > 0 && compUnit.Usings.Last().GetLastToken() == token)
+                if (compUnit.Usings.Count > 0 && compUnit.Usings.Last().SemicolonToken == token)
```

Comment:
Why this is changed? Is there a test cover this specific change?

Reply:
SemicolonToken is the last token of a using declaration.

Reply:
Sure, what I'm curious about is whether this change would cause any change in observable behavior in certain scenario (e.g. erroneous code)

Reply:
I can't think of a case where it would make a difference, but if there's a concern I can revert this part.